cmd/cstor-volume-grpc/api/handler.go

```diff
@@ -0,0 +1,80 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/golang/glog"
+
+	"github.com/openebs/maya/pkg/client/generated/cstor-volume-grpc/v1alpha1"
+	"github.com/openebs/maya/pkg/util"
+	"golang.org/x/net/context"
+)
+
+// sudo $ISTGTCONTROL snapdestroy vol1 snapname1 0
+// sudo $ISTGTCONTROL snapcreate vol1 snapname1
+
+// constants
+const (
+	VolumeGrpcListenPort = 7777
+	CmdSnapCreate        = "SNAPCREATE"
+	CmdSnapDestroy       = "SNAPDESTROY"
+	//IoWaitTime is the time interval for which the IO has to be stopped before doing snapshot operation
+	IoWaitTime = 10
+	//TotalWaitTime is the max time duration to wait for doing snapshot operation on all the replicas
+	TotalWaitTime = 60
+)
+
+//CommandStatus is the response from istgt for control commands
+type CommandStatus struct {
+	Response string `json:"response"`
+}
+
+//APIUnixSockVar is unix socker variable
+var APIUnixSockVar util.UnixSock
+
+// Server represents the gRPC server
+type Server struct {
+}
+
+// RunVolumeSnapCommand generates response to a RunCommand request
```

Comment:
I'd expect methods for create, destroy, list etc. not only mega method that does a big switch/if right?